server/routes/slack/slack_events.py

```
@@ -120,37 +127,65 @@ def slack_events():
+                                else:
+                                    # Non-background response (e.g. auth error) — just send normally
+                                    client.send_message(
+                                        channel=channel, 
+                                        text=response_text, 
+                                        thread_ts=thread_ts
+                                    )
+                                
+                                if trigger_background and sent_msg:
                                     thinking_ts = sent_msg.get('ts')
                                     
                                     logger.info(f"Processing @Aurora mention in channel {channel}, thread {final_thread_ts}: {text[:100]}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)